src/App.js

```diff
@@ -18,12 +19,17 @@ function App() {
   function addTodo (newTodo) {
     // setTodoList ([...todoList, newTodo])
     setTodoList(prevTodoList => ([ ...prevTodoList, newTodo ]))
-  }   
+  }  
+  
+  function removeTodo(id) {
+    setTodoList(todoList.filter((todo) => todo.id !== id));
```

Comment:
Nitpick (doesn't require changes):
Instead of filtering the `todoList`, you can use a callback function to filter the `prevTodoList` (similar feedback I left in the last assignment about adding a todo):
```diff
- setTodoList(todoList.filter((todo) => todo.id !== id));
+ setTodoList(prevTodoList => prevTodoList.filter(todo => todo.id !== id));
```